src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -1162,9 +1165,20 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         }
         const allFeatures = this.imageMap.getAllFeatures();
         const nextFeature = allFeatures.find((f) => f.get("id") === (nextRegionId));
-        const polygon = nextRegionId.split(",").map(parseFloat);
-        this.addToSelectedRegions(nextRegionId, nextFeature.get("text"), polygon, LabelCategory.Text);
-        this.redrawFeatures(allFeatures);
+        if (nextFeature) {
```

Comment:
we will not include these change right?

Reply:
this one is for the keyboard accessibility, but I don't why it still here, since I believed I already pushed it in last branch